spec/spec_helper.rb

```diff
@@ -2,6 +2,8 @@
 require 'mysql2'
 require 'timeout'
 require 'yaml'
+require 'fiber'
```

Comment:
It sounds like there's no need to raise the minimum Ruby for this import, but I definitely would be comfortable raising the baseline for this.

Reply:
Yeah I don't think it's needed.

IMHO 2.0 is really really old and people who are still on it don't deserve time. I now require 2.5 across all my projects. But that is totally independent of this PR.